packages/create-react-admin/templates/common/gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
I wondered whether to name this file `.gitignore.template` to make it more explicit